app/build.gradle

```diff
@@ -74,6 +77,17 @@ android {
 
 apply from: '../gradle/dependencies.gradle'
 
+ktlint {
+    version = "0.36.0" // https://github.com/pinterest/ktlint/issues/764
+    android = true
+    enableExperimentalRules = false
+    verbose = true
+    disabledRules = [
+            "import-ordering",
```

Comment:
Any reason for disabling this? Would it make too many changes?

Reply:
The reason is that the IDE does not yet support the official ordering. So when you auto-format a file in the IDE, the CI will fail.